### PR TITLE
Fixed issue #17745: Incomplete instructions to export response data to SPSS using Python

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/views/admin/export/spss_view.php [hunk 1]
```diff
@@ -103,7 +103,7 @@
         <ol>
             <li><?php eT("Download the data and the syntax file.");?></li>
             <li><?php eT("Open the syntax file in SPSS in Unicode mode.");?></li>
-            <em><?php eT("The next step (editing the 'FILE=' line) is only necessary when you have selected a SPSS version without Python. If you selected the version for SPSS with the Python plugin / Essentials, just save the syntax and the data file in the same folder. The full path will be automatically detected when you run the syntax."); ?></em>
+            <em><?php eT("The next step (editing the 'FILE=' line) is only necessary when you have selected a SPSS version without Python. If you selected the version for SPSS with the Python plugin / Essentials, just save the syntax and the data file in the same folder. If you use Python 3 you need to edit the syntax file: replace the line \"begin program.\" with \"begin program PYTHON3.\". The full path will be automatically detected when you run the syntax."); ?></em>
```
Comment (olleharstedt) [inline application/views/admin/export/spss_view.php:106]: Any idea how `\"` will interact with the script that scans for localized strings?

Comment (gabrieljenik) [reply application/views/admin/export/spss_view.php:106]: No idea... shall we wait or maybe replace by encoded html?

Comment (olleharstedt) [reply application/views/admin/export/spss_view.php:106]: Can we find another instance in the code base? Else, yeah, the encoded char might be better.

Comment (gabrieljenik) [reply application/views/admin/export/spss_view.php]: Replaced double quotes by single quotes :)

**Pull request merged**